app/ui/browser/views/navbar/location.jsx

```diff
@@ -221,7 +238,9 @@ class Location extends Component {
             onFocus={this.handleURLBarFocus}
             onBlur={this.handleURLBarBlur}
             onChange={this.props.onLocationChange}
-            onKeyDown={this.handleURLBarKeyDown}
+            onKeyDown={(e) =>
```

Comment:
event handler function signature should start with `e` as the first argument IMO